docs/src/installation_upgrade.md

```diff
@@ -236,6 +236,179 @@ When versions are not directly upgradable, the old version needs to be
+replicas to ensure the cluster doesn't lose any data.
+
+If you want to retain the old behavior, you need to set explicitly:
+
+```yaml
+spec:
+   ...
+   stopDelay: 30
+```
+
+And, **after** the upgrade has completed, specify `smartShutdownTimeout`:
+
+```yaml
+spec:
+   ...
+   stopDelay: 30
+   smartShutdownTimeout: 15
+```
+
+!!! Important
+    This change has been backported to all supported minor releases. As a
+    result, it will also be available in versions 1.20.3 and 1.19.5.
+
+#### Delay for PostgreSQL startup
+
+Until now, [the `startDelay` parameter](instance_manager.md#startup-liveness-and-readiness-probes)
+was set to 30 seconds, and CloudNativePG was using this parameter as the initial
+delay for the Kubernetes liveness probe. Given that all the supported Kubernetes
+releases provide [startup probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes),
+version 1.21 has adopted this approach as well (`startDelay` is now
+automatically divided into periods of 10 seconds of duration  each).
+
+!!! Important
+    In order to add the `startupProbe`, each pod needs to be restarted.
+    As a result, when you upgrade the operator, a one-time rolling
+    update of the cluster will be executed even in the online update case.
+
+Despite the recommendations to change and tune this value, almost all the cases
+we have examined during support incidents or community issues show that this
+value is left unchanged. Given that this parameter influences the startup of
+a PostgreSQL instance, a low value of `startDelay` would cause Postgres
+never to reach a consistent recovery state and be restarted indefinitely.
+
+For this reason, `startDelay` has been [raised by default to 3600 seconds](https://github.com/cloudnative-pg/cloudnative-pg/commit/4f4cd96bc6f8e284a200705c11a2b41652d58146),
+the equivalent of 1 hour.
+
+You can achieve similar behavior using the startup probe instead of
+liveness probes with an initial delay by explicitly setting:
+
+```yaml
+spec:
+   ...
+   startDelay: 30
+```
+
+!!! Important
+    This change has been backported to all supported minor releases. As a
+    result, it will also be available in versions 1.20.3 and 1.19.5.
+
+#### Delay for PostgreSQL switchover
+
+Up to now, [the `switchoverDelay` parameter](instance_manager.md#shutdown-of-the-primary-during-a-switchover)
+was set by default to 40000000 seconds (over 15 months) to simulate a very long
+interval.
+
+The [default value has been lowered to 3600 seconds](https://github.com/cloudnative-pg/cloudnative-pg/commit/9565f9f2ebab8bc648d9c361198479974664c322),
+the equivalent of 1 hour.
+
+If you want to retain the old behavior, you need to set explicitly:
+
+```yaml
+spec:
+   ...
+   switchoverDelay: 40000000
+```
+
+!!! Important
+    This change has been backported to all supported minor releases. As a
+    result, it will also be available in versions 1.20.3 and 1.19.5.
```

Comment:
same comment as before. Under "upgrading to 1.21"

Reply:
Ohh ... so we should reverse everything. Upgrade from 1.19.4 or before, 1.20.2 or before, or any previous version?

Reply:
Not sure what the best approach is. Was thinking something like below.
What do you think?

```
### Upgrading to 1.21, 1.20.3, 1.19.5

New defaults are being set for various things in these three releases.
Justification about users leaving the old defaults on, despite our recommendations.
Explanation of the one-time rolling upgrade.

#### New functionality in 1.21 only

#### Upgrading to 1.21 from 1.19.5 or 1.20.3

#### stopDelay parameter

#### startDelay

#### 
```

Reply:
Agree. Check the new proposal please.